src/components/views/rooms/RoomSublist.tsx

```diff
@@ -589,6 +590,16 @@ export default class RoomSublist extends React.Component<IProps, IState> {
                             >
                                 { _t("Show previews of messages") }
                             </StyledMenuItemCheckbox>
+                            <br />
+                            <br />
```

Comment:
Let's use CSS for styling rather than using semantic HTML elements

Reply:
Thanks for reviewing! Do you have an example for me? I'd gladly change that

Reply:
I would remove the two `<br />` and edit `_RoomSublist.scss` to create the spacing here using `margin-top` or any other relevant CSS property that'll create the effect you're looking for

Reply:
Sounds good, I think changing Rooms u list will change all elements and I just needed space between the two check boxes and the bottom - they are all part of the list I think, or maybe I'll just remove the button from the list...